utils/roboSender/roboSchema.ts

```diff
@@ -33,6 +33,9 @@ export let roboSchema = new mongoose.Schema({
         required: true,
         default: 'pending'
     },
+
+    // Array de dispositivos registrados en pacienteApp
+    device_id: Array,
```

Comment:
```suggestion
    device_id:  { type: string[], required: false },
```
Queda bien tipado y en el caso de un email/sms no se crea el campo porque no es requerido.

Reply:
Modificado